helm/superset/values.yaml

```diff
@@ -216,6 +217,18 @@ ph-ee-engine:
     requests:
       cpu: "100m"
       memory: "256M"
+    ingress:
```

Comment:
Are these keys added to helm/ph-ee-engine/values.yaml with  default values?

Reply:
path: "/"
      backend:
        service:
          name: ph-ee-connector-ams-mifos
          port:
            number: 80 
            I will remove these from superset and add to base chart later.